k8s-bench/tasks/find-application-level-failures/artifacts/eval-app-pod.yaml

```diff
@@ -0,0 +1,19 @@
+# Pod for running the `eval-app.py` script
+apiVersion: v1
+kind: Pod
+metadata:
+  name: eval-app-pod
+  namespace: app-level-fails-test
+spec:
+  containers:
+  - name: eval-app-executor
+    image: python:3.9-slim-buster
+    command: ["python"]
+    args: ["/etc/config/eval-app.py"]
```

Comment:
lets rename `eval-app.py` to something more real. may be call it `app.py`. 

lets fix this here and in all the other places in the PR.